introduction-to-datasets.md

```diff
@@ -0,0 +1,103 @@
+
+To define a dataset, Kite minimally requires a [URI](#uris) and a [schema](#schemas).
+
+## URIs
+
+Kite identifies datasets by URI. The URI you provide tells Kite how and where to store data. For example, a dataset created with the URI `dataset:hdfs:/user/cloudera/datasets/movies` is stored in HDFS.
+
+The default storage _scheme_ (as opposed to a _[schema](#schemas)_, described below) is Hive. If you omit the scheme in your URI, your dataset is stored in Hive. For example, the URI `movies` is equivalent to `dataset:hive:/movies`. A dataset you create with this URI is stored as a table in Hive's `default` database.
+
+Dataset URIs cannot contain periods (.).
+
+You can create datasets in Hive, HDFS, HBase, or as local files. See [Dataset and View URIs]({{site.baseurl}}/URIs.html).
+
+## Schemas
+
+A schema defines the field names and datatypes for a dataset. Kite relies on an Apache Avro schema definition for each dataset. For example, this is the schema definition for a table using four columns from the `movies` dataset.[<sup>1</sup>](#notes)
+
+```json
+{
+  "type":"record",
+  "name":"Movie",
+  "namespace":"org.kitesdk.examples.data",
+  "fields":[
+    {"name":"id","type":"int"},
+    {"name":"title","type":"string"},
+    {"name":"releaseDate","type":"string"},
+    {"name":"imdbUrl","type":"string"}
+  ]
+}
+```
+
+The goal is to get the schema into `.avsc` format. The following links provide examples for inferring schemas from data files or Java classes.
+
+| Java API                                                      | Command Line Interface |
+| --------                                                      | ---------------------- |
+| [Inferring a schema from a Java Class][api-schema-from-class] | [Inferring a schema from a Java class][cli-schema-from-class] |
+| [Using the schema of an Avro data file][api-schema-from-data] | [Inferring a schema from a CSV file][cli-schema-from-csv] |
+
+[api-schema-from-class]: {{site.baseurl}}/Inferring-a-Schema-from-a-Java-Class.html
+[api-schema-from-data]: {{site.baseurl}}/Inferring-a-Schema-from-an-Avro-Data-File.html
+[cli-schema-from-class]: {{site.baseurl}}/cli-reference.html#obj-schema
+[cli-schema-from-csv]: {{site.baseurl}}/cli-reference.html#csv-schema
+
+## Configuration Options
+
+While you need only a URI and schema to create a dataset, you can enhance the performance of your dataset with additional configuration.
+
+### Partitions
+
+Partitions define logical categories for data storage. For example, you might most often retrieve your data using time-based queries. You can define a partitioning strategy by year, month, day, and hour. When you look for data from January 8, 2015 between 8:00 and 9:00 a.m., your search engine only has to look in the data partition `/2015/1/8/8`. By using partitions that correspond to your most common queries, your data searches run more quickly.
```

Comment:
We always need to be careful with time examples to avoid confusion about how time zones are handled. The time we use for the partitioning is _always_ in UTC. Your example will be correct if the local time zone lines up with UTC. Otherwise, looking for data between 8 and 9 (local time) usually looks in partition /2015/1/8/16 to account for the zone. I suggest we avoid this entirely by removing the hour part of the example.